servers/pollux/default.nix

```diff
@@ -0,0 +1,59 @@
+{ modulesPath, inputs, config, pkgs, ... }:
+let
+  vs = config.vault-secrets.secrets;
+in {
```

Comment:
I don't think we use secrets anywhere here

Reply:
we sure don't, deleted